docs/core/tools/dotnet-nuget-sign.md

```diff
@@ -0,0 +1,162 @@
+---
+title: dotnet nuget sign command
+description: The dotnet nuget sign command signs all the packages matching the first argument with a certificate. The certificate with the private key can be obtained from a file or from a certificate installed in a certificate store by providing a subject name or a thumbprint.
+author: heng-liu
+ms.date: 07/07/2021
+---
+# dotnet nuget sign
+
+**This article applies to:** ✔️ .NET 6.0.0-preview.5 SDK and later versions
+
+## Name
+
+`dotnet nuget sign` - Signs all the NuGet packages matching the first argument with a certificate.
+
+## Synopsis
+
+```dotnetcli
+dotnet nuget sign [<package-path(s)>]
+    [-o|--output <OUTPUT DIRECTORY>]
+    [--certificate-path <PATH>]
+    [--certificate-store-name <STORENAME>]
+    [--certificate-store-location <STORELOCATION>]
+    [--certificate-subject-name <SUBJECTNAME>]
+    [--certificate-fingerprint <FINGERPRINT>]
+    [--certificate-password <PASSWORD>]
+    [--hash-algorithm <HASHALGORITHM>]
+    [--timestamper <TIMESTAMPINGSERVER>]
+    [--timestamp-hash-algorithm <HASHALGORITHM>]
+    [--overwrite]
+    [-v|--verbosity <LEVEL>]
+
+dotnet nuget sign -h|--help
+```
+
+## Description
+
+The `dotnet nuget sign` command signs all the packages matching the first argument with a certificate. The certificate with the private key can be obtained from a file or from a certificate installed in a certificate store by providing a subject name or a SHA-1 fingerprint.
+
+## Arguments
+
+- **`package-path(s)`**
+
+  Specifies the file path to the package(s) to be signed. Multiple position arguments can be passed in to verify multiple packages.
+
+## Options
+
+- **`-o|--output`**
+
+  Specifies the directory where the signed package should be saved. If this option is not specified, by default the original package is overwritten by the signed package.
+
+- **`--certificate-path <PATH>`**
+
+  Specifies the file path to the certificate to be used in signing the package.
+
+> [!NOTE]
+> This command currently supports only `PKCS12 (PFX)` files that contain the certificate's private key.
```

Comment:
Indentation to make it appear under the list item.

```suggestion
  > [!NOTE]
  > This command currently supports only `PKCS12 (PFX)` files that contain the certificate's private key.
```

Reply:
Thanks! Fixed.